platform-sdk/platform-apps/tests/AddressBookTestingTool/src/main/java/com/swirlds/demo/addressbook/AddressBookTestingToolState.java

```diff
@@ -298,23 +311,25 @@ private boolean validateTestScenario() {
         if (platform == null) {
             throw new IllegalStateException("platform is null, init has not been called.");
         }
-        logStakingBehavior.set(false);
-        final int testScenario = testingToolConfig.testScenario();
+        logWeightingBehavior.set(false);
+        final String testScenario = testingToolConfig.testScenario();
         try {
             logger.info(DEMO_INFO.getMarker(), "Validating test scenario {}.", testScenario);
             switch (testScenario) {
```

Comment:
Is there a reason that `testScenario` can't be an enum?

Reply:
turned into an enum of AddressBookTestScenario